### DISCUSSION TITLE
Accessing System File Directory?

### DISCUSSION THREAD

**Discussion by trymeouteh** [waydroid/waydroid#416] [Q&A]
In Wine, your able to access your Linux file directory by creating essentially adding symlinks. Most Wine prefixes come with the Z drive as the Linux root directory.

Is it possible to do the same with Waydroid?

**Comment by ACR-Jeff** [comment]
~/.local/share/waydroid is the location of the waydroid root directory

**Reply by trymeouteh** [reply]
Are you able to access your linux root directory within the Android OS in Waydroid?

**Reply by longqzh** [reply]
No, you can't

**Comment by Ammar64** [comment]
How do I access the /system directory from linux I want to write to it I can't write to the one in `/var/lib/waydroid/rootfs/system` because it's an overlay

**Reply by baptx** [reply]
For the read-only file system like `/system`, you have to write in `/var/lib/waydroid/overlay/system/`, see:
https://github.com/waydroid/waydroid/issues/870

**Reply by Smoker1** [reply]
How can I drop Files into it on Steam Deck? Have a App Installed, but seems like FTP messed something up. Need to Access...
~/.local/share/waydroid/data/media/0/
